set.mm

```diff
@@ -220226,8 +220331,8 @@ operation is a permutation group (group consisting of permutations), see
         UOUL $.
 
       $( The symmetric group on a pair is the symmetric group S_2 consisting of
-         the identity and the transposition.  This theorem is also valid if the
-         elements are identical: then it collapses to theorem ~ symg1bas .
+         the identity and the transposition.  In the case that both elements
+         are identical, S_2 collapses to S_1, see theorem ~ symg1bas .
```

Comment:
"S_2 collapses to S_1" is incorrect. It's no different from, say, "3 collapses to 6".

Reply:
OK, I will revert this change (proposed by Gérard...).

Reply:
Comment revised.